pyproject.toml

```diff
@@ -60,7 +60,7 @@ examples = [
     "matplotlib",
 ]
 test = [
-    "array_api_strict>=2",
+    "array_api_strict<2.4",
```

Comment:
Do we not want to keep the `>=2` if there was a reason to have it? If not then good to go from me

Reply:
If someone were to install it, then it would install `2.3.1`. @Saransh-cpp you implemented that bound. Do you feel strongly?

Reply:
Ah, I just realised why I did this. We use the `api_version` flag to adhere to a specific array API version -

https://github.com/glass-dev/glass/blob/4349efed157136a458248d6da3d64801a42a0ec6/tests/conftest.py#L72

and this flag was introduced in `2.0.0`, so yes, we should keep `>=2`.